.github/CODEOWNERS

```diff
@@ -1,3 +1,7 @@
+# By default, this whole repo should be reviewed by the product architect
+/* @elastisys/product-architect
```

Comment:
is there some visibility/permission issue with the team?

Reply:
I think `@elastisys/product-architect` needs to be explicitly added as "Collaborator" with write access to the repo.

![image](https://github.com/user-attachments/assets/bd6b6d41-0a94-472a-a153-1ff991c3461d)

Reply:
@Xartos could you fix this as well? :slightly_smiling_face:

Reply:
Done